MobProgrammingFacilitorsGuide_English.md

```diff
@@ -254,7 +254,7 @@ Emotions are often something not brought to work. But they are powerful indicato
 
 ## More Information
 
-[Download the full guidebook](http://mobprogrammingguidebook.com)  
+[Download the full guidebook](https://mobprogrammingguidebook.xyz)  
```

Comment:
Looks like it's back up at https://www.mobprogrammingguidebook.com/, and with https.

```suggestion
[Download the full guidebook](https://www.mobprogrammingguidebook.com/)  
```